react/components/EXPERIMENTAL_Select/index.js

```diff
@@ -258,6 +260,8 @@ Select.propTypes = {
   value: PropTypes.oneOfType([OptionShape, OptionsShape]),
   /** Max height (in _px_) of the selected values container */
   valuesMaxHeight: PropTypes.number,
+  /** If its options are initially shown */
+  defaultMenuIsOpen: PropTypes.bool,
```

Comment:
shouldn't it be added to `defaultProps` with a value of `false`?

Reply:
As its not required I think it doesn't matter, no?

Reply:
It's just a way of guaranteeing, because it will arrive undefined, but the code understands that it is a falsy.

Reply:
I'll add it anyway